site/app/templates/grading/electronic/Status.twig

```diff
@@ -316,7 +316,7 @@
                 {% endif %}
             </div>
             {# Graph Setup for Autograding #}
-            <div id="Autograding Histogram" class="tabcontent">
+            <div id="Autograding Score Histogram" class="tabcontent">
```

Comment:
These are both invalid ids as per defined by the [HTML 5 spec](https://www.w3.org/TR/html5/dom.html#element-attrdef-global-id), namely that an ID can be any string of characters except for no spaces.

Personally, I think all IDs should be kebab-case.

Reply:
made a note to fix when I revisit with instructor feedback
I already have some tweaks to make. I am new to html and its specs, I will try to improve in future and apologize for the inconvenience.

Reply:
Another note to this is that while it's technically allowed to have `#` and `.` in a id identifier, it's really not suggested as you then have to remember to always escape them when referencing them in CSS/JS and that forgetting to escape will make it refer to potentially different elements/scope altogether.